### DISCUSSION TITLE
Redirect

### DISCUSSION THREAD

**Discussion by sajkofff** [rejetto/hfs#508] [Q&A] [answered]
Hello.  Is it possible to redirect to another IP address if the main one is unavailable?

**Comment by rejetto** [comment, marked as answer]
possibly, but this is not something hfs can do, you have to work at a different level

**Comment by sajkofff** [comment]
understood thanks

**Comment by PH5HP** [comment]
Hi,

If you have your own domain name you could redirect in html and if your server is not available then display a message. Maybe there is more possible with a php script on your own domain. :-)

Kind regards,
Henk - https://fileserver.ph5hp.nl (if my computer is online)

**Comment by sajkofff** [comment]
Thanks for the advice. Although my knowledge is too little to implement this.
